app/src/main/java/com/sun/ise/helper/FragmentToolbar.kt

```diff
@@ -0,0 +1,42 @@
+package com.sun.ise.helper
+
+import android.view.MenuItem
+import androidx.annotation.IdRes
+import androidx.annotation.MenuRes
+import androidx.annotation.StringRes
+
+class FragmentToolbar(
+    @IdRes val resId: Int,
+    @StringRes val title: Int,
+    @MenuRes val menuId: Int,
+    val menuItems: MutableList<Int>,
+    val menuClicks: MutableList<MenuItem.OnMenuItemClickListener?>
+) {
+    class Builder {
```

Comment:
e k cần dùng builder cũng được
có thể tạo ra đốit uợgn bằng cách
class FragmentTooolbar(resId = , title = , . ..)